### DISCUSSION TITLE
Feature request:  Add entity showing Node location of VM

### DISCUSSION THREAD

**Discussion by k3vyn32** [dougiteixeira/proxmoxve#120] [Feature request] [answered]
Hi, this is a really useful integration - thank you.

I would be keen to see which promox node within my cluster a particular vm is running.
This is an option within the summary of a vm on the promox dashboard, so assume it could be queried/intergrated.

Thank you

**Comment by dougiteixeira** [comment]
At the moment I don't have the time to invest in these improvements, but I'm happy to merge them if you want to contribute.

**Comment by dougiteixeira** [comment, marked as answer]
It was added in version 2.0.0.b0 (beta).

If you can give feedback on how it works I would appreciate it.

**Comment by k3vyn32** [comment]
HI Doug,

Yep looks good, moved a few vms around the cluster and HA reports the
location.

Thank you

Kevyn
